### PR TITLE
Javascript modal

### PR DESCRIPTION
**Pull request by Aiden-Peace300**
image
image

### CODE REVIEW

File: javascript-modal/index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Javascript Modal</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="wrapper">
+      <button type="button" class="openButton">Open Modal</button>
```
Comment (shawnkost) [inline javascript-modal/index.html:11]: Class names should be using `kebab-cased`, not `camelCased`

File: javascript-modal/index.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    </div>
+
+    <div class="container">
+      <div class="surveryQuestionPopupWindow overlay hidden">
```
Comment (shawnkost) [inline javascript-modal/index.html:15]: Class names should be using `kebab-cased`, not `camelCased`

File: javascript-modal/index.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        <div class="popup">
+          <h1 class="surveyQuestion">Would you like to take a survey?</h1>
```
Comment (shawnkost) [inline javascript-modal/index.html:17]: Class names should be using `kebab-cased`, not `camelCased`

**Review by shawnkost — APPROVED** [review]
Approved, but check out my comments!

File: javascript-modal/index.html (hunk 1 above)
Comment (Aiden-Peace300) [reply javascript-modal/index.html]: Updated

File: javascript-modal/index.html (hunk 2 above)
Comment (Aiden-Peace300) [reply javascript-modal/index.html]: Updated

File: javascript-modal/index.html (hunk 3 above)
Comment (Aiden-Peace300) [reply javascript-modal/index.html]: Updated

**Pull request merged**